packages/next-on-pages/src/buildApplication/processVercelFunctions/prerenderFunctions.ts

```diff
@@ -13,6 +13,8 @@ import {
 } from '../../utils';
 import { cliWarn } from '../../cli';
 
+type Opts = Pick<ProcessVercelFunctionsOpts, 'functionsDir' | 'outputDir'>;
```

Comment:
not a huge deal but I wonder if we could have a meaningful name for this type instead of `Opts` 🤔